include/solvers/initial_conditions.h

```diff
@@ -81,6 +81,7 @@ namespace Parameters
   public:
     InitialConditions()
       : uvwp(dim + 1)
+      , cahn_hilliard(2)
```

Comment:
This does not need to be put there. You can set-up the initial conditions directly into the solver class like in the other approaches. Or otherwise, generate another class for CAHN.